velox/exec/GroupingSet.cpp

```diff
@@ -413,14 +413,18 @@ void GroupingSet::initializeGlobalAggregation() {
   lookup_->reset(1);
 
   // Row layout is:
-  //  - null flags - one bit per aggregate,
+  //  - alternating null flag, intialized flag - one bit per flag, one pair per
+  //                                             aggregation,
   //  - uint32_t row size,
   //  - fixed-width accumulators - one per aggregate
   //
   // Here we always make space for a row size since we only have one row and no
   // RowContainer.  The whole row is allocated to guarantee that alignment
   // requirements of all aggregate functions are satisfied.
-  int32_t rowSizeOffset = bits::nbytes(aggregates_.size());
+
+  // Allocate space for the null and initialized flags.
+  int32_t rowSizeOffset =
+      bits::nbytes(aggregates_.size() * RowContainer::kNumAccumulatorFlags);
```

Comment:
Hi @kevinwilfong , seems the flags don't consider for `sortedAggregations_` and `distinctAggregations_`? cc @Yuhta @mbasmanova